docs/src/pages/components/backdrop/backdrop.md

```diff
@@ -16,3 +16,12 @@ In its simplest form, the backdrop component will add a dimmed layer over your a
 ## Example
 
 {{"demo": "pages/components/backdrop/SimpleBackdrop.js"}}
+
+## Unstyled
+
+The backdrop also comes with the unstyled package.
```

Comment:
Should we be more descriptive? Same for https://next.material-ui.com/components/slider/#unstyled-slider

```suggestion
The backdrop also comes with the unstyled package.
It has the same API minus the styles.
```

Reply:
Aslo, should the header include the name of the component?

Reply:
I've reverted this text, main reason is that we have the API of both components in the end of the page. I'd say stating what the purpose of the unstyled component is is more important when showing it, the API is anyway available. I've updated all unstyled sections to be aligned.